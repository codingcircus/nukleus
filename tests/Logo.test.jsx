import React from 'react';
import renderer from 'react-test-renderer';
import Logo from 'Logo'; // eslint-disable-line import/no-unresolved, import/extensions, import/no-extraneous-dependencies

test('Renders Logo without crashing', () => {
  const component = renderer.create(
    <Logo
      duration={3000}
      href="/playground/"
      shade="light"
      title="kununu" />
  );
  expect(component.toJSON()).toMatchSnapshot();
});
