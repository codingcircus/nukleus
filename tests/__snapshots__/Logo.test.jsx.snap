exports[`test Renders Logo without crashing 1`] = `
<a
  className="logo light"
  onClick={[Function]}
  style={Object {}}>
  <h1
    className="sr-only">
    kununu
  </h1>
  <span
    aria-hidden="true"
    className="starSpinner light"
    style={
      Object {
        "transform": "rotate(0deg)",
      }
    } />
</a>
`;
